tests/components/homematicip_cloud/test_hap.py

```diff
@@ -121,3 +150,80 @@
     await hap.async_reset()
 
     assert len(hass.config_entries.async_forward_entry_unload.mock_calls) == 8
+
+
+@patch("homeassistant.helpers.temperature.display_temp", Mock(return_value=12))
+@patch("homeassistant.components.weather.WeatherEntity.state_attributes", new={})
+@patch(
+    "homeassistant.helpers.storage.Store._async_callback_stop_write",
```

Comment:
Can we use the available `hass_storage` fixture to mock storage?

Reply:
removed this